maintain.js

```diff
@@ -1,25 +1,345 @@
-/* Copyright © 2021 Ríona Stokes and other contributors, MIT License. */
-'use strict'
+module.exports = () => {
 
-const Fs = require('fs')
+  const Filehound = require('filehound')
+  const Path = require('path')
```

Comment:
separate built-in modules and external modules - easier to see external deps
https://github.com/senecajs/seneca/blob/master/seneca.js#L5

Reply:
Resolved with commit 8ffd82fb6a786b28be7c2e95fd85e53de929fd9c